# Lock the SWS toolchanger
---
industrial_msgs/TriState status
